src/gamemodes.py

```diff
@@ -117,7 +117,7 @@ def set_default_totem_chances(self):
     # Here so any game mode can use it
     def lovers_chk_win(self, evt, var, rolemap, mainroles, lpl, lwolves, lrealwolves):
         winner = evt.data["winner"]
-        if winner is not None and winner.startswith("@"):
+        if winner in Win_Stealer:
```

Comment:
This is wrong. Lovers not winning with fool is a special case due to how easy it is to set up that win condition with lovers. Lovers can (and should) win with other win stealing roles like monster, demoniac, piper, etc.

Also, the name of that variable is all sorts of wrong. Make it CamelCase without underscores.

Reply:
I'll look into it later.

Also, which variable? Win_Stealer? It's been like that since I added globals for cats. I can change it but not as part of this PR.